src/routes.js

```diff
@@ -1,12 +1,11 @@
 import Route from './components/app/route'
+import Header from './containers/redditClone'
```

Comment:
this variable should be called `RedditClone` not header (because very soon it will be incorrect)